graphdatascience/tests/integrationV2/procedure_surface/cypher/conftest.py

```diff
@@ -2,27 +2,50 @@
 from pathlib import Path
 from typing import Generator
 
+import dotenv
 import pytest
 from testcontainers.core.container import DockerContainer
 from testcontainers.core.waiting_utils import wait_for_logs
 from testcontainers.neo4j import Neo4jContainer
 
 from graphdatascience import QueryRunner
+from graphdatascience.query_runner.gds_arrow_client import GdsArrowClient
 from graphdatascience.query_runner.neo4j_query_runner import Neo4jQueryRunner
 
 
 @pytest.fixture(scope="package")
-def gds_plugin_container(logs_dir: Path, inside_ci: bool) -> Generator[Neo4jContainer, None, None]:
+def gds_plugin_container(
+    logs_dir: Path, inside_ci: bool, tmp_path_factory: pytest.TempPathFactory
+) -> Generator[Neo4jContainer, None, None]:
     neo4j_image = os.getenv("NEO4J_DATABASE_IMAGE", "neo4j:enterprise")
 
+    dotenv.load_dotenv("graphdatascience/tests/test.env")
+    GDS_LICENSE_KEY = os.getenv("GDS_LICENSE_KEY")
+
     neo4j_container = (
         Neo4jContainer(
             image=neo4j_image,
         )
         .with_env("NEO4J_ACCEPT_LICENSE_AGREEMENT", "yes")
         .with_env("NEO4J_PLUGINS", '["graph-data-science"]')
+        .with_env("NEO4J_gds_arrow_enabled", "true")
+        .with_env("NEO4J_gds_arrow_listen__address", "0.0.0.0:8491")
+        .with_exposed_ports(8491)
     )
 
+    if GDS_LICENSE_KEY is not None:
+        license_dir = tmp_path_factory.mktemp("gds_license")
+        license_dir.chmod(0o755)
```

Comment:
this is needed for the license to be accessible for the neo4j user in the docker container?